src/components/play/services/scoring/ChooseSeveralScoring.ts

```diff
@@ -20,8 +34,8 @@ const mark = (component: any, attempt: ComponentAttempt<ChooseSeveralAnswer>) =>
 
     attempt.correct = true;
```

Comment:
Can we try moving this up to the top of the scope. I think 'hoisting' moves it up anyway at compile time.

Reply:
https://www.w3schools.com/js/js_hoisting.asp